erpnext_documentation/www/docs/user/manual/en/manufacturing/bill-of-materials.md

```diff
@@ -18,7 +18,7 @@ To make accurate Purchase Requests, you must always maintain correct BOMs.
 To access the BOM list, go to:
```

Comment:
@rohitwaghchaure in all files, please run grammarly first to filter basic mistakes.
And follow the format, definition first, then navigation, then steps.
Screenshots should have some white space in width and show the top navigation part in the screenshot.